src/js/views/PollList.vue

```diff
@@ -81,7 +83,7 @@
 						</template>
 					</PollItem>
 				</template>
-			</TransitionGroup>
+			</RecycleScroller>
 
 			<IntersectionObserver v-if="showMore"
```

Comment:
Or backporting the chunking to stable6

Reply:
I currently try to limit the usage of components before the vue3 migration. But since the stable-6 is an dead end, it is up to you. Might be a nice test implementation for further optimisations of the master. 

I first wanted to implement chunked loading from the backend, but sorting the list would break the concept or would make it even more complex.

So it is up to you.

Reply:
I just see, that you plan it to merge into master. So no. Not yet. We can keep it open or retry later.

Reply:
Of course you can target stable-6, but please get a consence between you and come-nc, since he is on the way to the release.

Reply:
Alright, I'll target stable6 then. As you said it it will be a nice test :)